src/Standards/PSR2/Sniffs/Namespaces/UseDeclarationSniff.php

```diff
@@ -142,7 +142,7 @@ public function process(File $phpcsFile, $stackPtr)
             return;
         }
 
-        $next = $phpcsFile->findNext(T_WHITESPACE, ($end + 1), null, true);
+        $next = $phpcsFile->findNext([T_WHITESPACE, T_PHPCS_IGNORE, T_COMMENT], ($end + 1), null, true);
```

Comment:
You may want to use `Tokens::$EmptyTokens` instead for completeness.

Reply:
Thanks. Updated!